packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -1234,6 +1234,15 @@ function updateSuspenseComponent(
           NoWork,
           null,
         );
+
+        if (enableProfilerTimer && workInProgress.mode & ProfileMode) {
+          // Fiber treeBaseDuration must be at least as large as the sum of children treeBaseDurations.
+          // Otherwise the profiler's onRender metrics will be off,
+          // and the DevTools Profiler flamegraph will visually break as well.
```

Comment:
Could you rewrite this comment in terms of what it means semantically instead of what will break if it goes wrong?

"treeBaseDuration is the sum of all the children durations" would be sufficient